Ferry.java

```diff
@@ -1,11 +1,54 @@
+import java.util.ArrayList;
+
 /**
  * A ferry provides a one-way journey between two islands. It
  * has a ferry code and information about both the source and
  * the destination island
  *
- * @author (your name)
+ * @author (20067052)
  * @version (a version number or a date)
  */
 public class Ferry {
+    private int ferry_ID;
```

Comment:
This is Java, not Python. The variable names should be in `camelCase`.

Reply:
noted and fixed